src/test/java/org/assertj/core/internal/lists/Lists_satisfiesAt_Test.java

```diff
@@ -0,0 +1,58 @@
+package org.assertj.core.internal.lists;
+
+import static org.assertj.core.data.Index.atIndex;
+import static org.assertj.core.error.ShouldNotBeNull.shouldNotBeNull;
+import static org.assertj.core.test.TestData.someInfo;
+import static org.assertj.core.util.Lists.newArrayList;
+
+import java.util.List;
+import java.util.function.Consumer;
+
+import org.assertj.core.api.AssertionInfo;
+import org.assertj.core.api.Assertions;
+import org.assertj.core.data.Index;
+import org.assertj.core.internal.ListsBaseTest;
+import org.junit.Test;
+
+public class Lists_satisfiesAt_Test extends ListsBaseTest {
+
+  private final AssertionInfo info = someInfo();
+  private final Consumer<String> requirements = str -> Assertions.assertThat(str).isEqualTo("Luke");
+  private final Index index = atIndex(1);
+  private final List<String> actual = newArrayList("Leia", "Luke", "Yoda");
+
+  @Test
+  public void should_pass_if_actual_match_requirements() {
+    lists.satisfiesAt(info, actual, requirements, index);
```

Comment:
I don't get it: what is "lists" and what is "actual"?
It seems to me that it should look like:

```java
assertThat(actual).satisfiesAt(info, requirements, index);
```

Reply:
that's a test for `Lists` which is the class that actually implements the assertion business logic

Reply:
ok, thanks for the explanation.